Sources/PointFree/FeatureFlags.swift

```diff
@@ -12,9 +12,12 @@ extension Array where Element == Feature {
   ]
 
   func hasAccess(to feature: Feature, for user: Database.User?) -> Bool {
-
-    return
-      feature.isEnabled
-        || (feature.isAdminEnabled && user?.isAdmin == .some(true))
+    return self
+      .first(where: { $0.name == feature.name })
+      .map {
+        $0.isEnabled
+          || ($0.isAdminEnabled && user?.isAdmin == .some(true))
+      }
+      ?? false
```

Comment:
ok, the whole `[Feature].hasAccess(to:)` API is so whack. I dont know what I was thinking when I wrote it. here I just did a minimal amount of work to make it sensible, but we should revamp it sometime soon.